pulsar-broker/src/main/java/org/apache/pulsar/broker/admin/v3/Transactions.java

```diff
@@ -168,6 +209,7 @@ public void getTransactionMetadata(@Suspended final AsyncResponse asyncResponse,
                                        @DefaultValue("false") boolean authoritative,
                                        @PathParam("mostSigBits") String mostSigBits,
                                        @PathParam("leastSigBits") String leastSigBits) {
+        checkTransactionCoordinatorEnabled();
```

Comment:
Looks like we should catch this method exception.

Reply:
It's ok, like `validateTopicName`